documentation/chatmeleon-swagger.yaml

```diff
@@ -228,7 +326,60 @@ components:
           type: string
           format: date-time
           description: The timestamp when the user account was last updated.
+        conversationsIds:
+          type: array
+          items:
+            type: string
+            format: uuid
+            description: List of conversation IDs the user is part of
+        seenMessageIds:
+          type: array
+          items:
+            type: string
+            format: uuid
+            description: List of message IDs the user has seen
 
+    Account:
+      type: object
+      properties:
+        id:
```

Comment:
Does the API client need to know about this schema? Do we use this in our API now or in the future?

Reply:
Yea we dont need the Account schema, we can remove it